src/main/java/com/frcteam3255/components/swerve/SN_SuperSwerve.java

```diff
@@ -250,6 +256,11 @@ public void setModuleStates(SwerveModuleState[] desiredModuleStates, boolean isO
 		// well
 		SwerveDriveKinematics.desaturateWheelSpeeds(desiredModuleStates, swerveConstants.maxSpeedMeters);
 
+		for (int i = 0; i < 8; i += 2) {
+			swerveDesiredStates[i] = desiredModuleStates[i / 2].angle.getRadians();
+			swerveDesiredStates[i + 1] = desiredModuleStates[i / 2].speedMetersPerSecond;
+		}
```

Comment:
what are these lines doing?

Reply:
Those are a remnant of when AdvantageScope only accepted doubles in arrays. I'm pretty sure AdvantageScope added logging swerveModuleStates directly so I can test that out and then remove this